.agents/skills/seo-geo/scripts/seo_audit.py

```
@@ -0,0 +1,151 @@
+import urllib.request
+import urllib.parse
+import re
+import time
+import sys
+
+
+def fetch_url(url: str, timeout: int = 30) -> tuple:
+    """Fetch URL and return (content, headers, load_time)"""
+    try:
+        start = time.time()
+        req = urllib.request.Request(url, headers={"User-Agent": "SEO-Audit/1.0"})
```

Ruff (0.15.1)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)